QUICKSTART/QUICKSTART.md

```diff
@@ -0,0 +1,258 @@
+
+# Quickstart for VSCODE-CODE-COVERAGE
+
+**Summary**:  This is the QUICKSTART.md document to help users get started with the `ryanluker.vscode-coverage-gutters` "*Coverage Gutters*" extension for VS Code.
+
+## Terminology
+
+*Coverage Gutters* depends uses two files.  It's easy to get the files mixed up and we need to make sure that everone is on the same page when we talk about these files.
+
+<details>
+<summary>Coverage File</summary>
+
+-----
+
+  * LCOV or XML file containing detailed coverage data
+
+* *Coverage Gutters* supports LCOV, CLOVER, COBERTURA, and JACOCO coverage files
+
+* Generation of the coverage file is language specific and is beyond the scope of this extension.
+
+</details>
+
+<details>
+<summary>Coverage Report</summary>
+
+-----
+
+  * HTML file(s) that provides a nice way to view coverage data.
```

Comment:
These dropdown md sections are pretty cool.
Above I mentioned the extra space at the front of the first item, but I am thinking that is a part of the format now 😓.